.github/workflows/ci.yaml

```
@@ -0,0 +1,65 @@
+name: Build lint and check formatting
+
+on:
+  pull_request:
```

Por padrão o on pull request roda no opened, synchronize e reopened, acho bacana colocar o ready_for_review também, já que ela não roda quando ta em draft.

```suggestion
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
```